### ISSUE TITLE
Process is stuck after error is thrown in fetchAll c++ fn

### ISSUE THREAD

**Issue by kirrg001** [ibmdb/node-ibm_db#846]
Hi there!

I have a problem with a use case. My process get's stuck and I cannot kill it when I execute this combination:

```
connection.prepare(stmt, (err, stmtObject) => {
    ....
    stmtObject.execute(function (subErr, result) {
       ...
       const args = [null, null, callback];
       
       try {
         result.fetchAll(...args);
       } catch(err) {
          // error is thrown and catched, this works
          // but then the process is kinda stuck if I try to kill it
       }
    })
})
```

You get the following raised error, which is expected:

```
TypeError: ODBCResult::FetchAll(): 1 or 2 arguments are required. The last argument must be a callback function.
```

The error is thrown in your c++ lib
https://github.com/ibmdb/node-ibm_db/blob/822f2dcfddeb3f8438723d37cbd061f3635d3fb0/src/odbc_result.cpp#L455

I think a return is missing?

Thanks
Kati

**Comment by bimalkjha** [comment]
@kirrg001 This issue is fixed in ibm_db@2.8.2. Thanks.

**Comment by kirrg001** [comment]
Thanks. I can confirm that this is solved now
